src/averageAges.js

```diff
@@ -15,11 +16,15 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const men = century
+    ? people.filter((p) => p.sex === 'm' && Math.ceil(p.died / 100) === century)
+    : people.filter((p) => p.sex === 'm');
```

Comment:
When there is only one argument for an arrow function we usually don't put brackets. Please apply this change for all occurrences. [See detailed explanation here](https://mate-academy.github.io/style-guides/javascript.html#84-if-your-function-takes-a-single-argument-and-doesnt-use-braces-omit-the-parentheses-otherwise-always-include-parentheses-around-arguments-for-clarity-and-consistency)
```suggestion
  const men = century
    ? people.filter(p => p.sex === 'm' && Math.ceil(p.died / 100) === century)
    : people.filter(p => p.sex === 'm');
```